application-xpoll-test/application-xpoll-test-pageobjects/src/main/java/org/xwiki/xpoll/test/po/InPreparationStatusViewPage.java

```diff
@@ -28,12 +31,15 @@
     @FindBy(xpath = "//div[@class = 'xform']/dl/dd")
     public WebElement pollDescription;
 
-    @FindBy(xpath = "//div[@class = 'xform']//dd[1]")
-    public WebElement pollProposals;
```

Comment:
not sure why you've renamed it. Before it had a pattern with the other variables.